server/src/pages/settings/view.js

```diff
@@ -0,0 +1,72 @@
+const reactruntime = require("../../reactruntime");
+const React = require("react");
+
+class Head extends React.Component {
+
+  render() {
+    return (
+      <reactruntime.HeadTemplate {...this.props}>
+        <script src={this.props.staticLink("/static/js/settings-bundle.js")} async></script>
+        <link rel="stylesheet" href={this.props.staticLink("/static/css/settings.css")} />
+      </reactruntime.HeadTemplate>
+    );
+  }
+
+}
+
+class Body extends React.Component {
+  render() {
+    return (
+      <reactruntime.BodyTemplate {...this.props}>
+        <div className="column-space full-height">
+          <a className="button close-preferences" href="/shots"></a>
+          <div>
+            { this.renderAccountInfo() }
+          </div>
+        </div>
+      </reactruntime.BodyTemplate>
+    );
+  }
+
+  renderAccountInfo() {
+    let defaultAvatar = this.props.staticLink('/static/img/default-profile.svg');
+    let info;
+    if (this.props.accountInfo) {
+      info = (
+        <div className="account-info">
+          <img src={this.props.accountInfo.avatar || defaultAvatar } height="100" width="100" />
+            <div className="info-container">
+              <p>{this.props.accountInfo.nickname || this.props.accountInfo.email}</p>
+              <a className="account-buttons manage-accounts" href="">Manage Accounts</a>
+              <a className="account-buttons disconnect" href="" onClick={ this.onClickDisconnect.bind(this) }>Disconnect</a>
+            </div>
+        </div>
+      );
+    } else {
+      info = (
+        <div className="account-info">
+          <img src={ defaultAvatar } height="100" width="100" />
+            <div className="info-container">
+              <p>Guest Account</p>
```

Comment:
Is the Guest Account notion in the wireframes? I don't have an opinion, but @youwenliang might.